code-challenges.test.js

```diff
@@ -1,69 +1,316 @@
+
+// // a) Create a test with expect statements for each of the variables provided.
 
 const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
+// // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// // Expected output: [72, 81, 90, 99, 108]
+
+
+// // b) Create the function that makes the test pass.
 
+const timesThree = numbersArray1.map(value => {
+  return value * 3
+})
 
-// b) Create the function that makes the test pass.
+console.log(timesThree)
 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+//1//
+// // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const num1 = 15
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 const num2 = 0
-// Expected output: "0 is divisible by three"
+// // Expected output: "0 is divisible by three"
 const num3 = -7
-// Expected output: "-7 is not divisible by three"
+// // Expected output: "-7 is not divisible by three"
+
+//----- MY TEST -------
+
+describe("create a function call it outcome", () => {
+
+    it("use if statement to check if num1 is divisible by three with no remainder, if statement is true then log num1 is divisible by three, if statement is false then log num2 is not divisible by three", () => {
+ 
+      expect(outcome(num1)).toEach("15 is divisible by three")
+    })
+})      
+
+describe("create a function call it outcome", () => {
+
+    it("use if statement to check if num2 is divisible by three with no remainder, if statement is true then log num2 is divisible by three, if statement is false then log num2 is not divisible by three", () => {
+ ""
+      expect(outcome(num2)).toEach("0 is divisible by three")
+    })
+}) 
+
+describe("create a function call it outcome", () => {
+
+    it("use if statement to check if num3 is divisible by three with no remainder, if statement is true then log num3 is divisible by three, if statement is false then log num2 is not divisible by three", () => {
+ 
+      expect(outcome(num3)).toEach("-7 is not divisible by three")
+    })
+})
+
+
+
+// // b) Create the function that makes the test pass.
+
+
+//----- I created a functiion called outcome
+
+const outcome = () => {
+
+//----- I used a if statement to return num1 as (`${num1} is divisible by three`) if num1 is divisible by 3 with no remainder
+
+     if (num1 % 3 == 0) {
+   
+   
+       return (`${num1} is divisible by three`)
+      } 
+      else {
+   
+//----- If else, return (`${num1} is not divisible by three`)
+
+       return (`${num1} is not divisible by three`)
+       }
+    } 
```

Comment:
Your logic looks good!  Instead of accessing the specific test variable, try to utilize parameters in your function to allow it to be more dynamic.  That means you only need to write one function and can test any possible variable.